### DISCUSSION TITLE
I've created a TiddlyWiki Lezer Parser

### DISCUSSION THREAD

**Discussion by BurningTreeC** [TiddlyWiki/TiddlyWiki5#8795] [General]
Hello,

for my CodeMirror 6 editor I went down the rabbit hole and created a TiddlyWiki5 - to Lezer converter which integrates with the CodeMirror parsing system and adds highlighting for TiddlyWiki5 syntax.

Now what I'm doing internally is parsing the whole input with:

```
$tw.wiki.parseText(fullInput,{parseAsInline: false})
```

But I'd like to have more input, a more complex parse tree, with the widgets and such.

How do I achieve that?

**Comment by pmario** [comment]
Can you link to your code?

**Reply by BurningTreeC** [reply]
When I'm back at my desk ok?

I've found a way in the meantime
